bin/fk.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+source $FANTASTIC_ROOT/bin/wrappers.sh
```

Comment:
Is `_fk_wrapper_open` going to be exported and available from command line  ?

Reply:
Yes. Since fk.sh gets `source`d. 

I did a quick search and I don't think there is another way to call _fk_wrapper_open without basically exporting it or having it all in here. 

That is why I did `_fk...`., because generally people don't call _cmd in shell.

FYI FClone, Fpr are also available to everyone.

Reply:
:cry: rip, ok I guess we can refactor those later then